app/globals.css

```
@@ -100,3 +100,197 @@ html {
+  border-collapse: collapse;
+  width: 100%;
+  table-layout: fixed;
+  margin: 0.5rem 0;
+}
+
+.prose-editor .ProseMirror th,
+.prose-editor .ProseMirror td {
+  border: 1px solid #e2e2e2;
+  padding: 0.4rem 0.6rem;
+  min-width: 2rem;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)